src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -306,10 +306,10 @@
          :on-done (fn [{:keys [success? get-parsed-response]}]
                     (swap! state dissoc :blocker :editing?)
                     (if success?
-                      (do (swap! state dissoc :redacted?)
-                          ((:on-rename props) name)
+                      (do ((:on-rename props) name)
                           ((@refs "methodDetailsViewer") :clear-redacted-snapshot)
-                          (swap! state assoc :loaded-config (get-parsed-response) :blocker nil))
+                          (utils/multi-swap! state (assoc :loaded-config (get-parsed-response))
+                                                   (dissoc :redacted?)))
```

Comment:
do you also need to `dissoc :blocker` here?

Reply:
Nope, that's already getting `dissoc`ed above.